cns/restserver/ipam.go

```diff
@@ -87,6 +94,7 @@
 	if err = service.releaseIPConfig(podInfo); err != nil {
 		statusCode = NotFound
 		returnMessage = err.Error()
+		logger.Errorf("releaseIPConfigHandler releaseIPConfig failed because %v", returnMessage)
```

Comment:
Again can you add the context, otherwise we wont be able to figure out this error message is for what request